bin/spectacle/init.js

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const prompts = require('prompts');
+const parse = require('./args');
```

Comment:
This is unused and a circular import (which is technically OK). I think we just need to remove it?

Reply:
Ah, I didn't realize I left these here. Thanks.